.github/workflows/shared-release-pull-request-library-java.yml

```diff
@@ -76,10 +76,10 @@ jobs:
         id: sha
         run: echo "short-sha=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
 
-  create-draft-release:
-    name: Create Draft Release
+  pr-release-notes:
```

Comment:
This was named wrong, does not create a draft release, just add release notes to the PR description